### DISCUSSION TITLE
Is it possible to read maddy emails using Gmail?

### DISCUSSION THREAD

**Discussion by lts20050703** [emersion/maddy#718] [Q&A] [answered]
Originally I plan to forward all emails to my gmail account, but after reading the forwarding guide I realize that it's not something I should do.

Is there a way I can read maddy email using gmail?

Anyways thank you so much for making this Max Mazurov, we really appreciate it ❤️

**Comment by cxvqo** [comment, marked as answer]
I don't think Gmail Web client offers this feature, you should look into using [Thunderbird](https://www.thunderbird.net/en-US/) (or a fork of it, known as [Betterbird](https://www.betterbird.eu/)).
On mobile Gmail however, you do have the option to use a custom email server. ("Add another account" -> "Other")
